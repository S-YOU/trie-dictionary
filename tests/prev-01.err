nodes: 2
nodes: 4
nodes: 6
nodes: 8
nodes: 10
nodes: 8
nodes: 6
nodes: 8
nodes: 6
nodes: 7
nodes: 6
