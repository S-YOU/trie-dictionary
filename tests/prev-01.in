insert aaaaaab
find b
prev 0 1 6
find b
prev 1 1 5
find b
prev 2 1 4
find b
prev 3 1 3
find b
delete 0
delete 2
prev 4 1 2
find b
delete 4
prev 5 1 1
find b
delete 6
find b
