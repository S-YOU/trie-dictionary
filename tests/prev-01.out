word number: 0
NO
word number: 1
NO
word number: 2
NO
word number: 3
NO
word number: 4
NO
deleted: 0
deleted: 2
word number: 5
NO
deleted: 4
word number: 6
YES
deleted: 6
NO
